urlchecker/client/check.py

```diff
@@ -48,7 +48,11 @@ def main(args, extra):
         sys.exit("Error %s does not exist." % path)
 
     # Parse file types, and excluded urls and files (includes absolute and patterns)
-    file_types = args.file_types.split(",")
+    file_types = []
```

Comment:
Why is this necessary?

Reply:
Also, if the user puts a singe character expansion, this wouldn't be included since it's == 1.

Reply:
true, but as far as I know the shortest file type is 2 characters like `.c` or `.*`. How would a one sign file type look like?

Reply:
Try doing `ls .` and you'll see all files in the present working directory.

Reply:
yes but doing `--files-types  .` doesn't seem correct. `.` is not a files type. ofc the user can use `*` but do we really need to account  for that? btw the reason for this change is imo related to https://github.com/urlstechie/urlchecker-action/issues/76

Reply:
yes I can add it in a separate one, I only did it  here because we were waiting on the asyncio code so I thought I ship them together.